src/main/docker/docker_root/swarmclean.py

```diff
@@ -308,10 +316,15 @@ def query_single_value(self, query: str, arg_values={}):
     return self.do_query(query, arg_values)[0][0]
 #end class AlfrescoDB
 
-@dataclass
+
+def parse_http_timestamp(timestamp: str) -> datetime:
+    return parsedate(timestamp).astimezone()
+
```

Comment:
Is it correct for me to assume this method takes in the http-header string for the timestamp as: "Wed, 21 Oct 2015 07:28:00 GMT" and converts it into a datetime Object?

Reply:
indeed